doc/man3/SSL_CTX_load_verify_locations.pod

```diff
@@ -63,7 +65,16 @@ used.
 
 =head1 NOTES
 
-If B<CAfile> is not NULL, it points to a file of CA certificates in PEM
+The CA certificates specified by calling the above functions are used
+as trust anchors for peer certificate verification purposes unless overridden
+by L<SSL_CTX_set0_verify_cert_store(3)>, L<SSL_CTX_set1_verify_cert_store(3)>,
+by L<SSL_set0_verify_cert_store(3)>, or L<SSL_set1_verify_cert_store(3)>.
+They are used as trust anchors for
+building the chain of the own client/server certificate unless overridden
```

Comment:
```suggestion
building the client os server's own certificate chain, unless overridden
```

Reply:
changed to "client or server's own certificate chain"